### PR TITLE
feat(ODX-283): Bootstrap [DOCKER]

### PR DESCRIPTION
**Pull request by QuentinMllt**

### CODE REVIEW

File: samples/dsm/akeneo-dsm/front-end/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
```
Comment (gwendalaubert) [inline samples/dsm/akeneo-dsm/front-end/README.md:7]: Not sure about this as this is a dockerised project. Either change to "In the container dsm-front" or change the command to a docker-compose run / make command one.

File: samples/dsm/akeneo-dsm/back-end/.env [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="file:/srv/app/var/dsm_apps.db"
+
+###> Application version ###
+APPLICATION_VERSION=1.1.0
```
Comment (gwendalaubert) [inline samples/dsm/akeneo-dsm/back-end/.env:11]: Change this to 1.0.0 as this is the first version of this app ?

**Review by gwendalaubert — CHANGES_REQUESTED** [review]
Change the docker-compose.yml and every other file in common with front-end and back-end. Create a Makefile in the root directory that will call front-end and back-end Makefiles.
Create a common README.md in the root directory but keep the ones in the children directories.

File: samples/dsm/akeneo-dsm/front-end/README.md (hunk 1 above)
Comment (QuentinMllt) [reply samples/dsm/akeneo-dsm/front-end/README.md:7]: There is a README task ODX-287 in the next sprint to fix all the README.

**Review by gwendalaubert — APPROVED** [review]

**Pull request merged**